packages/react-components/src/AddressSmall.tsx

```diff
@@ -6,6 +6,9 @@ import type { AccountId, Address } from '@polkadot/types/interfaces';
 import React from 'react';
 import styled from 'styled-components';
 
+import ParentAddress from '@polkadot/react-components/ParentAddress';
+import { toShortAddress } from '@polkadot/react-components/util';
```

Comment:
```suggestion
import ParentAddress from './ParentAddress';
import { toShortAddress } from './util';
```

We are in the same package.

Reply:
✅